codegen/core/src/test/scala/com/lightbend/akkasls/codegen/ModelBuilderSuite.scala

```diff
@@ -30,7 +30,39 @@ import scala.collection.mutable
 
 import com.lightbend.akkasls.codegen.TestData.fullyQualifiedName
 
-class ModelBuilderSuite extends munit.FunSuite {
+object ModelBuilderSuite {
+  trait Config {
+    def label: String
+    def registry: ExtensionRegistry
+    def descriptorSetPath: String
+  }
+  object CodegenAnnotationConfig extends Config {
+    override val label: String = "CodegenAnnotation"
+    override val descriptorSetPath: String = "descriptor-sets"
+
+    def registry: ExtensionRegistry = {
+      val reg = ExtensionRegistry.newInstance()
+      reg.add(com.akkaserverless.codegen.Annotations.codegen)
+      reg.add(com.akkaserverless.Annotations.method)
```

Comment:
I initially thought that `method` was only used on the proxy side and didn't copied it here. 

It's surely used there because that's what tell the proxy how to create the views and eventing. Not clear why we need this in the build tests.